cli/asc.js

```diff
@@ -923,6 +983,56 @@ exports.main = function main(argv, options, callback) {
   }
 };
 
+const toString = Object.prototype.toString;
+
+function isObject(arg) {
+  return toString.call(arg).slice(8, -1) === "Object";
```

Comment:
```suggestion
  return toString.call(arg) === '[object Object]';
```
This shorter and faster

Reply:
Ah. Yeah. Usually the slice is for convenience and not speed. Are there other places where we can speed things up like this?